pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -905,6 +905,11 @@ func (p *criStatsProvider) addCadvisorContainerStats(
 	if memory != nil {
 		cs.Memory = memory
 	}
+
+	swap := cadvisorInfoToSwapStats(caPodStats)
```

Comment:
Any reason why we don't need something similar in cadvisor_stats_provider?

Reply:
It's already there AFAIU:
* https://github.com/kubernetes/kubernetes/blob/2d0a4f75560154454682b193b42813159b20f284/pkg/kubelet/stats/cadvisor_stats_provider.go#L130
* https://github.com/kubernetes/kubernetes/blob/2d0a4f75560154454682b193b42813159b20f284/pkg/kubelet/stats/cadvisor_stats_provider.go#L220